src/rootUtils.h

```diff
@@ -30,6 +30,78 @@ constexpr static auto metaTreeName = "podio_metadata";
  */
 constexpr static auto paramBranchName = "PARAMETERS";
 
+/**
+ * Names of the fields with the keys and values of the generic parameters for
+ * the RNTuples until map types are supported
+ */
+constexpr static auto intKey = "GPIntKeys";
```

Comment:
```suggestion
constexpr static auto intKeyName = "GPIntKeys";
```
mainly consistency since I think all previously existing branch/writer name strings have `Name` as their ending.